### DISCUSSION TITLE
Question about the ephemeris markers

### DISCUSSION THREAD

**Discussion by imariasantos** [Stellarium/stellarium#1730] [Q&A] [answered]
In the ephemeris tab from the astronomical calculations window, I can get the position of the Sun in the sky every 10 solar days for an entire year. When I do this, I expect to see a "figure 8" in the sky, since the Sun appears to follow the analemma. Yet, the ephemeris markers follow a circle. 

As seen from the Earth, with no ground or atmosphere:
![analemma ???](https://user-images.githubusercontent.com/57090282/121363442-3f75c880-c937-11eb-8da2-75f57d33d59f.png)

Then as I move through time, the position of the ephemeris markers move.
The view in May and in November from the same place at the same solar time.
![Screenshot from 2021-06-09 15-35-41](https://user-images.githubusercontent.com/57090282/121365040-992ac280-c938-11eb-96ea-4201adbc4c08.png)
![Screenshot from 2021-06-09 15-36-25](https://user-images.githubusercontent.com/57090282/121365055-9cbe4980-c938-11eb-8e98-6b34d70c41e6.png)

I think the markers from the ephemeris tool should show the position of the Sun in the sky, but that doesn't seem to be the case. What are the markers actually showing? And how do I plot the analemma on the sky?

**Comment by gzotti** [comment, marked as answer]
Activate the H.C. checkbox.

**Reply by imariasantos** [reply]
Thanks! It worked :)

Subsidiary question: What does "H.C." stand for?

**Reply by alex-w** [reply]
Tooltip for "H.C." checkbox is "Use horizontal coordinates"
